gettext_compile.py

```diff
@@ -8,6 +8,6 @@
 
 if __name__ == "__main__":
     do_compile("spyderlib")
-    do_compile("p_pylint", "spyderplugins")
-    do_compile("p_profiler", "spyderplugins")
-    do_compile("p_breakpoints", "spyderplugins")
+    do_compile("p_pylint", "spyderplugins/p_pylint")
+    do_compile("p_profiler", "spyderplugins/p_pylint")
+    do_compile("p_breakpoints", "spyderplugins/p_pylint")

```

Comment:
Erf, this is a bug indeed, but the problem I get is on the line 10: `do_compile("spyderlib")`. The following lines don't even get a chance to be run.